### PR TITLE
Authorized Proxies

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by hrkrshnn** [comment]
Is this related to the poly chain hack?

**Comment by chriseth** [comment]
Yes, at least this was the trigger for me to write it.

**Review by leonardoalt — DISMISSED** [review]

File: docs/security-considerations.rst [hunk 1]
```diff
@@ -193,6 +193,39 @@ Note that ``.send()`` does **not** throw an exception if the call stack is
...
+
+        function callOther(address _addr, bytes memory _payload) public
+                returns (bool, bytes memory) {
+            return proxy.callOther(_addr, _payload);
+        }
+        // Other functions and other functionality
+    }
+
+    // This is the full contract, it has no other functionality and no way
+    // to assume any privileges.
```
Comment (leonardoalt) [inline docs/security-considerations.rst]: Well, the privilege could be given by the target contract, assigning this as the owner for example. This is ofc the fault of the target contract, but not sure the sentence is precise in that case

Comment (chriseth) [reply docs/security-considerations.rst]: True. So maybe "has no other functionality and requires no privileges to work."?

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**